app-backend/export/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/azavea/spark:2.0.1
```

Comment:
What is this being used for? I don't think we should need a separate container for spark -- this will be running in EMR when deployed.